docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -638,6 +638,70 @@ Avaya
 Avaya bought Nortel's wired networks assets. So Avaya switches are, in effect, re-branded 
 Nortels. See <<_nortel,Nortel section>> of this document for configuration instructions. 
 
+802.1X with MAC Authentication bypass and VoIP
```

Comment:
MAC Authentication Bypass

I think, but I'm not sure, but that looks better to me